### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by lol219** [googleapis/google-api-python-client#1834] [General]
What is the latest build version? is it v1?

**Comment by H4ckerxx44** [comment]
I suppose you can look [here](https://github.com/googleapis/google-api-python-client/releases).

**Comment by lol219** [comment]
> I suppose you can look [here](https://github.com/googleapis/google-api-python-client/releases).

What

**Reply by H4ckerxx44** [reply]
You asked what the latest build version is, so I assumed you meant the latest release of the python client here and sent you the link to the latest release.
